.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+documentation:
+  - changed-files:
+      - any-glob-to-any-file: ['docs/**', 'mkdocs.yml']
+docker:
+  - changed-files:
+      - any-glob-to-any-file: ['**/Dockerfile', 'docker-compose.yml']
+element:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/elements/**']
+SQL:
+  - changed-files:
+      - any-glob-to-any-file: ['**/*.sql']
+python:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/python/**']
+javascript:
+  - changed-files:
+      - any-glob-to-any-file: ['**/*.js', '**/*.ts']
```

Comment:
This also seems more or less useless, at least for PRs (with the exception of Dependabot PRs, which will be automatically labeled anyways).

Reply:
Maybe? If you're laser-focused on Python stuff until you start full-time, you might be better off just focusing your attention on things with the `python` label rather than looking specifically to exclude JS things.

Reply:
Sure -- I buy that no tag generally means main PL changes.

Reply:
Should we remove the `javascript` label then?

Reply:
No; dependabot will just recreate it, and I often filter PRs to select those with the `dependabot` and `javascript` tags. It's useful in that context; probably not in any other ones though.